package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "@OnroerendErfgoed/vue_component_library",
+  "name": "@onroerenderfgoed/vue_component_library",
```

Comment:
ik denk niet dat deze mocht gebeuren, daardoor komen er import problemen in plannen enzo. ping @yannickkuypers

Reply:
https://docs.npmjs.com/creating-a-package-json-file#required-name-and-version-fields

Reply:
werkt wel, maar zou eigenlijk lowercase moeten zijn: String violates the pattern: '^(?:@[a-z0-9-*~][a-z0-9-*._~]*/)?[a-z0-9-~][a-z0-9-._~]*$'

Reply:
yh, maar hij gaat de github repo niet vinden omdat die hoofdletters heeft:
https://github.com/OnroerendErfgoed/ 
was gans gedoe ze, met yarn en de security tokens enzo

Reply:
Nieuw issue om dit eens verder te bekijken https://github.com/OnroerendErfgoed/vue_component_library/issues/92